src/Serilog.Sinks.AzureTableStorage/Sinks/AzureTableStorage/AzureBatchingTableStorageSink.cs

```diff
@@ -86,7 +88,9 @@ protected override void EmitBatch(IEnumerable<LogEvent> events)
 
                 _batchRowId++;
             }
-            _table.ExecuteBatch(operation);
+
+            _table.ExecuteBatchAsync(operation)
```

Comment:
Instead of waiting here, it's possible to override `PeriodicBatchingSink.EmitBatchAsync()` instead of `EmitBatch()` and just `await` the call.